src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -35,7 +35,7 @@ case class CsvRelation protected[spark] (
     useHeader: Boolean,
     delimiter: Char,
     quote: Char,
-    escape: Char,
+    escape: Character,
```

Comment:
Yes it make sense.
I also notice that the escape character only rendered when quote is set to null